pennylane/plugins/tests/conftest.py

```diff
@@ -79,6 +79,40 @@ def _skip_if(dev, capabilities):
     return _skip_if
 
 
+@pytest.fixture(scope="session")
+def skip_if_ops():
+    """Fixture to skip tests for unsupported operations."""
+
+    def _skip_if_ops(dev, ops):
+        """ Skip test if device does not support an operation. """
+
+        for op in ops:

```

Comment:
Minor point: Could we do `if op.__name__ not in dev.operations` here, to save all the `__name__`'s below?